lib/java/ydoc-server/src/main/java/org/enso/ydoc/polyfill/PolyfillBase.java

```diff
@@ -0,0 +1,21 @@
+package org.enso.ydoc.polyfill;
+
+import org.enso.ydoc.Polyfill;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Source;
+
+public abstract class PolyfillBase implements Polyfill {
+  private final String resourceName;
+
+  protected PolyfillBase(String resourceName) {
+    this.resourceName = resourceName;
+  }
+
+  @Override
+  public void initialize(Context ctx) {
```

Comment:
`final`. Unless you really want this method to be overridable.

Reply:
Possibly you may also make the method _package private` - it it is called only from the same package.

Reply:
How was this complaint resolved?

Reply:
Can't make it work with _package private_ with interfaces. Even when `Polyfill` is moved to `polyfill` subpackage it will still complain about being unable to access in `web` subpackage. I'm happy to be shown that I'm wrong but `javac` was quite stubborn about that.

I did forget about `final` though. Fixed.